src/WebJobs.Script/Rpc/LanguageWorkerChannel.cs

```diff
@@ -458,5 +460,21 @@ private bool IsTriggerMetadataPopulatedByWorker()
         {
             return !string.IsNullOrEmpty(_workerCapabilities.GetCapabilityState(LanguageWorkerConstants.RpcHttpTriggerMetadataRemoved));
         }
+
+        private RpcTraceContext ConstructAndGetTraceContext(string activityId, string traceStateString, IEnumerable<KeyValuePair<string, string>> tags)
+        {
+            RpcTraceContext traceContext = new RpcTraceContext
+            {
+                Traceparent = activityId ?? string.Empty,
+                Tracestate = traceStateString ?? string.Empty,
+            };
+
+            foreach (KeyValuePair<string, string> tag in tags ?? Enumerable.Empty<KeyValuePair<string, string>>())
+            {
+                traceContext.Attributes.Add(tag.Key, tag.Value);
```

Comment:
Like we talked about in person, handle cases for null or empty value here or in the proto file itself